unittests/files/keyusage.ts

```diff
@@ -1,3 +1,11 @@
+function assertType<T>(_x: T) {}
+
+const mockKey = {} as CryptoKey;
+assertType<Promise<JsonWebKey>>(crypto.subtle.exportKey('jwk', mockKey));
```

Comment:
Please also check what happens if the input is `'jwk' | 'pkcs8'`.

Reply:
100%. I'm on it.